.github/actions/combine-needs-status/action.yml

```diff
@@ -0,0 +1,15 @@
+---
+
+name: combine-needs-status
```

Comment:
Open for different naming suggestions @elastic/observablt-ci

Reply:
What do you think this other approach?

1) composite action will return a result (boolean)
2) Name of the composite actions will something like `are-dependent-jobs-success`?

IIUC, the current implementation will return errorlevel 1 if no matches with the `test` command.

I somehow feel we can use a boolean output to the composite action, so consumers can use the if condition like:


```

Reply:
I like the idea.

just writing this here to let it sink.

there is also the usecase for setting the "single status check".
this would become something like:

```yaml
  - id: is_success
    uses: elastic/apm-pipeline-library/.github/actions/are-dependent-jobs-success@current
    with: ${{ toJSON(needs) }}
  - run: ${{ steps.is_success.outputs.result }} # which should evaluate to `true` or `false` and have that status code `1` or `0`
```

which actually looks good is more expressive.


the `notify-build-status` takes the current job context and sends the correct status to slack dependening on the job result. So this would become:

```yaml
  - id: is_success
    uses: elastic/apm-pipeline-library/.github/actions/are-dependent-jobs-success@current
    with: ${{ toJSON(needs) }}
  - run: ${{ steps.is_success.outputs.result }} # which should evaluate to `true` or `false` and have that status code `1` or `0`
  - if: always()
    uses: elastic/apm-pipeline-library/.github/actions/notify-build-status@current
```

Reply:
https://github.com/elastic/apm-pipeline-library/pull/2126/commits/d208e86c20acaf5687d209714af2c572962c8dbe